lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -1139,8 +1156,8 @@ def format_coord(self, xd, yd):
             return ''
 
         if self.button_pressed in self._rotate_btn:
-            return 'azimuth=%d deg, elevation=%d deg ' % (self.azim, self.elev)
-            # ignore xd and yd and display angles instead
+            return 'azimuth={:.0f} deg, elevation={:.0f} deg '.format(
+                self.azim, self.elev)
```

Comment:
Bug here on fractional view angles

Reply:
Extracted to #12573. I misremembered - the crash was only for non-finite values.